docs/specification/new-spec/consensus.md

```diff
@@ -0,0 +1,197 @@
+# Tendermint Consensus
+
+Tendermint consensus is a distributed protocol executed by validator processes to agree on
+the next block to be added to the Tendermint blockchain. The protocol proceeds in rounds, where 
```

Comment:
`... to be added to a blockchain.` 

Theoretically it could be any blockchain. For example, Polkdadot might use Tendermint consensus.

Also, we should be consistent with naming and always call it `Tendermint Consensus`.